test/dynamo/test_aot_autograd_cache.py

```diff
@@ -0,0 +1,105 @@
+# Owner(s): ["module: dynamo"]
+
+import torch
+import torch._dynamo
+import torch._dynamo.test_case
+
+import torch._functorch._aot_autograd
+from torch._functorch._aot_autograd.autograd_cache import autograd_cache_hash
+from torch._functorch._aot_autograd.schemas import AOTConfig
+
+
+class AOTAutogradCachePicklerTests(torch._dynamo.test_case.TestCase):
+    def default_config(self):
+        return AOTConfig(
+            fw_compiler=None,
+            bw_compiler=None,
+            inference_compiler=None,
+            partition_fn=None,
+            decompositions={},
+            num_params_buffers=0,
+            aot_id=0,
+            keep_inference_input_mutations=False,
+            dynamic_shapes=True,
+            aot_autograd_arg_pos_to_source=None,
+            is_export=False,
+            no_tangents=False,
+            enable_log=False,
+        )
+
+    def _get_dynamo_output(self, fn, *args, **kwargs):
+        # Reset dynamo between runs
+        torch._dynamo.reset()
+        fx_graph = None
+
+        def compiler(gm, inputs, **kwargs):
+            nonlocal fx_graph
+            fx_graph = gm
+            return gm
+
+        g = torch.compile(fn, backend=compiler)
```

Comment:
you might want `fullgraph=True`? if there area graph breaks, `compiler()` will get called multiple times (and the nonlocal will get overwritten with whatever the last graph was)